scripts/extensions/sams/src/notifications/sets.ts

```diff
@@ -0,0 +1,41 @@
+import {getStore} from '../../src/store';
+import {loadSets, closeSetContentPanel} from '../../src/store/sets/actions';
+import {superdeskApi} from '../../src/apis';
+
+export function onSetCreated() {
+    const store = getStore();
+
+    store?.dispatch<any>(loadSets());
+}
+
+export function onSetUpdated(event: any) {
+    const {notify} = superdeskApi.ui;
+    const store = getStore();
+    const user_id = superdeskApi.session.getCurrentUserId();
+
+    store?.dispatch<any>(loadSets());
+    if (user_id !== event.detail.extra.user_id) {
+        const item_id = store?.getState().sets.selectedSetId;
```

Comment:
Use selectors when accessing a value from the redux state, i.e. `src/store/sets/selects.ts: getSelectedSetId`